geomstats/_backend/pytorch/__init__.py

```diff
@@ -66,6 +64,18 @@ def _raise_not_implemented_error(*args, **kwargs):
 vectorize = _raise_not_implemented_error
 
 
+def sin(x):
```

Comment:
I was thinking that for primitives like this (and all other unary functions), we might be better off using a decorator which wraps scalars, e.g.,
```python
def _box_scalar(function):
    @functools.wraps(function)
    def wrapper(x):
        if not torch.is_tensor(x):
            x = torch.tensor(x)
        return function(x)
    return wrapper

sin = _box_scalar(sin)
sinh = _box_scalar(sinh)
```
We could also make this a little more flexible by writing a "decorator factory" to support a variable number of arguments.

Reply:
Great idea, let me do this!